paper.md

```diff
@@ -51,40 +51,40 @@ In order to make the `shar` package as accessible for as many people as possible
 
 The two approaches differ in how they randomize the null model data, but both control for potential biotic processes by preserving the spatial structure of the data [@Plotkin2000; @Wiegand2014] and result in similar results.
-Finally, species-habitat associations are present if species are found in certain habitats in the data  more often than expected compared to the randomized null model data [@Plotkin2000; @Harms2001].
+Finally, species-habitat associations are present if species are found in certain habitats in the data more often than expected compared to the randomized null model data [@Plotkin2000; @Harms2001].
 Given the characteristics of the method, a positive association to one habitat inevitably leads to a negative association to at least one of the other habitats (and vice versa; Yamada et al. 2006).
 
 # How to use the package
 
 Analyzing species-habitat associations is straightforward with the `shar` package.
 Only two objects are needed to quantify species-habitat associations, namely a `spatstat` object that includes all object locations within the study area and a `raster` object with discrete habitat classes.
 
-However, all methods require "fully mapped data" in a sense that NA cells of the environmental data are allowed only if simultaneously these areas cannot accommodate any locations of the point pattern (e.g., a water body within a forest area).
+However, all methods require "fully mapped data" in the sense that NA cells of the environmental data are allowed only if simultaneously these areas cannot accommodate any locations of the point pattern (e.g., a water body within a forest area).
 This needs to be reflected in the observation window of the point pattern.
 For the torus translation method, no NA values are allowed at all.
 
-To randomize the environmental data, either `translate_raster()` or `randomize_raster` can be used.
+To randomize the environmental data, either `translate_raster()` or `randomize_raster()` can be used.
 For the former, the number of randomizations of the null model data is automatically determined by the number of rows and columns of the `raster` object.
-For the later, the number of randomizations must be specified using the `r_random` argument.
+For the later, the number of randomizations must be specified using the `n_random` argument.
 
 ```
 torus_trans <- translate_raster(raster = landscape_discrete)
 
 random_walk <- randomize_raster(raster = landscape_discrete, n_random = 39)
 ```
 
-Alternatively, to randomize the object locations, either `fit_point_process` or `reconstruct_pattern_*` can be used.
-In both cases, the number of randomization must be specified using the `r_random` argument.
+Alternatively, to randomize the object locations, either `fit_point_process()` or `reconstruct_pattern_*()` can be used.
```

Comment:
I didn't follow everything but didn't you regroup these `reconstruct_pattern_*()` functions under `reconstruct_pattern()`?

Reply:
There still is `reconstruct_pattern()` and `reconstruct_pattern_marks()` because they are slightly different ideas. So I didn't want to mix them. I will add this